bundles/AdminBundle/Resources/public/js/pimcore/object/classes/data/objectbricks.js

```diff
@@ -51,25 +51,28 @@ pimcore.object.classes.data.objectbricks = Class.create(pimcore.object.classes.d
 
     getLayout: function ($super) {
         $super();
-        
+
         this.specificPanel.removeAll();
-        this.specificPanel.add([
-            {
-                xtype: "numberfield",
-                fieldLabel: t("maximum_items"),
-                name: "maxItems",
-                value: this.datax.maxItems,
-                minValue: 0
-            },
-            {
-                xtype: "checkbox",
-                fieldLabel: t("border"),
-                name: "border",
-                checked: this.datax.border,
-            }
-        ]);
 
-        if(this.inCustomLayoutEditor) {
+        if(!this.inCustomLayoutEditor) {
+            this.specificPanel.add({
+                    xtype: "numberfield",
+                    fieldLabel: t("maximum_items"),
+                    name: "maxItems",
+                    value: this.datax.maxItems,
+                    minValue: 0
+            });
+        }
+
+        this.specificPanel.add({
+            xtype: "checkbox",
+            fieldLabel: t("border"),
+            name: "border",
+            checked: this.datax.border,
+        });
+
+
+        if (this.inCustomLayoutEditor) {
             this.specificPanel.add(new Ext.ux.form.MultiSelect({
```

Comment:
Is this field necessary at all in the custom layout editor?

Reply:
https://github.com/pimcore/pimcore/pull/12084 It is only implemented for custom layout, we may can think of another approach in an improvement.